client/components/takeQuiz/TakeQuiz.jsx

```diff
@@ -101,6 +107,8 @@ const TakeQuiz = (userId) => {
       return currentScore;
     })()
     .then(response => {
+      getFriendRankings(response);
+      getGlobalRankings(response);
       submitAnswers(response);
```

Comment:
Slightly knit-picky, but can you be a little more explicit with what the response here is?